README.md

```diff
@@ -1,2 +1,80 @@
-# holdshelf-poller
-Polls Sierra for items placed on a holdshelf. Notifies patrons.
+# Holdshelf Poller
+
+This is an app to poll Sierra for items placed on a holdshelf and notify those patrons.
+
+## Purpose
+
+This app connects to the Sierra database directly to identify items placed on a holdshhelf. The specific holdshelves and item holding locations are configurable. When it encounters an item on a holdshelf, it checks Redis to determine if the hold has already been processed. If it has not been processed, a call is made to the PatronServices Notify endpoint (TK).
+
+## Installation
+
+Via Docker:
+
+```
+docker image build -t holdshelf-poller:local .
+docker container run -e ENVIRONMENT=development holdshelf-poller:local
+```
+
+Or for development in OSX:
+```
+pyenv local 3.10
+python -m venv venv
```

Comment:
Personal preference but I think it's a little confusing to name the virtual environment venv

Reply:
What do you tend to name it? And you gitignore it via `*env`?

Reply:
It doesn't really matter because you're never submitting it but I tend to use 'testenv' or 'localenv' -- I think 'venv' is a little confusing because the command is also 'venv'. And yeah `*env` works to ignore it, although if you want to name any real file `<something>.env` you have to get a little fancier.